### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by DolineanuVlad**
Finished the 2nd assignment.

### CODE REVIEW

File: 30414/Dolineanu A. Vlad Ionut/Assignment 2/functions.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#ifndef FUNCTIONS_H_INCLUDED
+#define FUNCTIONS_H_INCLUDED
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#define ALLOC_NODE (node*)malloc(sizeof(node))
+
```
Comment (DolineanuVlad) [inline 30414/Dolineanu A. Vlad Ionut/Assignment 2/functions.h:8]: Is this define something I should be doing? I find it helps quite a lot but I don't know if it has other side effects

Comment (Alex-Cosma) [inline 30414/Dolineanu A. Vlad Ionut/Assignment 2/functions.h:8]: No, it's quite alright.

**Comment by DolineanuVlad** [comment]
Fixed those problems

**Comment by DolineanuVlad** [comment]
Updated with the second part of the assignment(qwewe!)

**Comment by Alex-Cosma** [comment]
Looks good

**Pull request merged**